helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -77,9 +77,10 @@ rules:
       - alertmanagers
       - alertmanagerconfigs
     verbs:
+      - get
+      - list
```

Comment:
Why do we need the list here ?

Reply:
Yes later on we list prometheus resources https://github.com/giantswarm/prometheus-meta-operator/pull/905/files#diff-f77bb5d20dbbf9e2633fb1c88f52e7fa098253d13887f4df3bd83f64688fd749R29